### PR TITLE
Mon 17987 enh ha upgrade to 23.04

### PR DESCRIPTION
**Pull request by cedricmeschin**
## Description

Major update for upgrade process for HA 22.04 to 23.04.
Simplifying process for more efficiency. No need to destroy cluster anymore.
If changes are validated, sames will be implemented on other upgrades HA process to 23.10 and newer.

Only EN version for instant

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [X] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Review by jaltmeyer — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-04.md [hunk 1]
```diff
@@ -343,10 +454,20 @@ pcs constraint colocation add master "ms_mysql-clone" with "vip_mysql"
 </TabItem>
 </Tabs>
 
-Recréez ensuite les contraintes qui empêchent les processus Centreon de s'exécuter sur les nœuds de base de données et vice-versa :
+Recréez ensuite la contrainte qui empêche les processus Centreon de s'exécuter sur les nœuds de la base de données et vice-et-versa.:
```
Comment (jaltmeyer) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-04.md:457]: Je vais chipoter mais c'est bien "vice-versa" : https://fr.wikipedia.org/wiki/Vice_versa

**Review by jaltmeyer — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-04.md (hunk 1 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-04.md:457]:
```suggestion
Recréez ensuite la contrainte qui empêche les processus Centreon de s'exécuter sur les nœuds de la base de données et vice-versa :
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-10.md [hunk 2]
```diff
@@ -343,10 +387,20 @@ pcs constraint colocation add master "ms_mysql-clone" with "vip_mysql"
 </TabItem>
 </Tabs>
 
-Recréez ensuite les contraintes qui empêchent les processus Centreon de s'exécuter sur les nœuds de base de données et vice-versa :
+Recréez ensuite la contrainte qui empêche les processus Centreon de s'exécuter sur les nœuds de la base de données et vice-et-versa.:
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-10.md:390]:
```suggestion
Recréez ensuite la contrainte qui empêche les processus Centreon de s'exécuter sur les nœuds de la base de données et vice-versa :
```

File: versioned_docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-04.md [hunk 3]
```diff
@@ -346,7 +456,17 @@ pcs constraint colocation add master "ms_mysql-clone" with "vip_mysql"
 Then recreate the Constraint that prevent Centreon Processes to run on Database nodes and vice-et-versa:
```
Comment (cg-tw) [inline versioned_docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-04.md:456]:
```suggestion
Then recreate the Constraint that prevent Centreon Processes to run on Database nodes and vice versa:
```

File: versioned_docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-10.md [hunk 4]
```diff
@@ -346,7 +390,17 @@ pcs constraint colocation add master "ms_mysql-clone" with "vip_mysql"
 Then recreate the Constraint that prevent Centreon Processes to run on Database nodes and vice-et-versa:
```
Comment (cg-tw) [inline versioned_docs/version-23.04/upgrade/centreon-ha/upgrade-from-22-10.md:390]:
```suggestion
Then recreate the Constraint that prevent Centreon Processes to run on Database nodes and vice versa:
```

**Review by jaltmeyer — APPROVED** [review]

**Pull request merged**